src/modules/sensors/als.c

```diff
@@ -146,8 +146,12 @@ static int capture(void *dev, double *pct, const int num_captures, char *setting
             }
         }
 
-        if (illuminance >= 0) {
-            pct[ctr++] = illuminance / max;
+        if (illuminance >= 1) {
+            if (illuminance != min) {
```

Comment:
Why using different branches for illuminance != min?

Reply:
If illuminance == min, then pct[ctr++] would be 0. I interpreted this as turning off the backlight entirely, which (at least to me) seems like an undesired behavior. So I figured if illuminance == min, to have it set to a minimal (but nonzero) brightness.

Reply:
I see; note that this is an ambient brightness measurement; ie: 0.0 means we are in a completely dark environment.
Then Clight uses this value to apply its backlight curve; this means 0.0 is a valid ambient brightness.
One can customize Clight regression points: https://github.com/FedeDP/Clight/blob/master/Extra/clight.conf#L94 to avoid having a 0.0 backlight on 0.0 ambient brightness.